keyboards/candykeys/candy87/keyboard.json

```diff
@@ -0,0 +1,892 @@
+{
+    "manufacturer": "Candykeys",
+    "keyboard_name": "Candy87",
+    "maintainer": "RarePotato8DE",
+    "board": "GENERIC_RP_RP2040",
+    "bootloader": "rp2040",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "extrakey": true,
+        "nkro": true,
+        "rgb_matrix": true
+    },
+    "matrix_pins": {
+        "cols": ["GP12", "GP28", "GP27", "GP26", "GP25", "GP24", "GP23", "GP22", "GP21", "GP20", "GP19", "GP18", "GP17", "GP16", "GP29", "GP15", "GP14", "GP13"],
+        "rows": ["GP0", "GP1", "GP2", "GP11", "GP9", "GP10"]
+    },
+    "processor": "RP2040",
+    "rgb_matrix": {
+        "animations": {
+            "breathing": true,
+            "cycle_all": true,
+            "digital_rain": true,
+            "jellybean_raindrops": true,
+            "pixel_rain": true,
+            "rainbow_moving_chevron": true,
+            "raindrops": true,
+            "starlight": true
+        },
+        "driver": "ws2812",
+        "layout": [
+            {"matrix": [0, 16], "x": 30, "y": 13, "flags": 9},
+            {"matrix": [3, 0], "x": 2, "y": 40, "flags": 9}
+        ]
+    },
+    "url": "https://candykeys.com/",
+    "usb": {
+        "device_version": "0.0.1",
+        "pid": "0x0001",
+        "vid": "0x434B"
+    },
+    "ws2812": {
+        "driver": "vendor",
+        "pin": "GP7"
+    },
+	"community_layouts": ["tkl_f13_ansi_split_bs_rshift", "tkl_f13_ansi_tsangan_split_bs_rshift", "tkl_f13_iso_split_bs_rshift", "tkl_f13_iso_tsangan_split_bs_rshift", "tkl_ansi_split_bs_rshift", "tkl_ansi_tsangan_split_bs_rshift", "tkl_iso_split_bs_rshift", "tkl_iso_tsangan_split_bs_rshift"],
```

Comment:
Could, remaining, compatible Community Layouts be added:
- tkl_ansi
- tkl_ansi_tsangan
- tkl_ansi_wkl
- tkl_ansi_wkl_split_bs_rshift
- tkl_iso
- tkl_iso_tsangan
- tkl_iso_wkl
- tkl_iso_wkl_split_bs_rshift
- tkl_f13_ansi
- tkl_f13_ansi_tsangan
- tkl_f13_ansi_wkl
- tkl_f13_ansi_wkl_split_bs_rshift
- tkl_f13_iso
- tkl_f13_iso_tsangan
- tkl_f13_iso_wkl
- tkl_f13_iso_wkl_split_bs_rshift

Reply:
just as option in the "community_layouts" array/list or do I also have to include the corresponding matrix layout definitions? Adding all of these layouts is a little overloaded, no?
Since WK Key will just be "ignored" if WKL is used. Same for "Fn" Key at the split right shift. Thanks for reply!